src/backend/distributed/utils/node_metadata.c

```diff
@@ -524,12 +538,31 @@ master_update_node(PG_FUNCTION_ARGS)
 	 */
 	if (WorkerNodeIsPrimary(workerNode))
 	{
+		/*
+		 * before acquiring the locks check if we want a background worker to help us to
+		 * aggressively obtain the locks.
+		 */
+		if (force)
+		{
+			handle = StartLockAcquireHelperBackgroundWorker(MyProcPid, lock_cooldown);
+			ereport(NOTICE, (errmsg("force the update to the node")));
```

Comment:
could be a bit more descriptive on what that means, e.g.: `cancelling pending transactions on node %d in %d milliseconds`

Reply:
actually, I think I will remove this.

I used this early in the development to make sure I was hitting the codepath in a quick and dirty way :). Given that the user invokes the function via `force => true` to get here I see little value in keeping this `NOTICE`